sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCUtils.scala

```diff
@@ -44,7 +44,7 @@ import org.apache.spark.util.NextIterator
 /**
  * Util functions for JDBC tables.
  */
-object JdbcUtils extends Logging {
+object JDBCUtils extends Logging {
```

Comment:
@MaxGekk . As you see the change on `PostgreSQL` dialect, this will break the downstream custom dialect. Shall we avoid to change this?